### PR TITLE
UI: Add audio mixer/transition toolbars

### PR DESCRIPTION
**Pull request by cg2121**
### Description
Adds toolbars to audio mixer/transition docks

Screenshot from 2022-07-26 18-44-04

### Motivation and Context
Make UI consistent.

### How Has This Been Tested?
Made sure buttons worked as expected.

### Types of changes
- Tweak (non-breaking change to improve existing functionality)

### Checklist:
- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

**Comment by cg2121** [comment]
Fixed merge commit

**Review by Warchamp7 — APPROVED** [review]

**Comment by cg2121** [comment]
I've now combined this with #5128 because they cause merge conflicts with each other.

File: UI/forms/OBSBasic.ui [hunk 1]
```diff
@@ -2244,6 +2215,75 @@
...
+    <string>menuIconSmall</string>
+   </property>
+  </action>
+  <action name="actionTransitionMenu">
+   <property name="icon">
+    <iconset resource="obs.qrc">
+     <normaloff>:/res/images/dots-vert.svg</normaloff>:/res/images/dots-vert.svg</iconset>
+   </property>
+   <property name="text">
+    <string>TransitionMenu</string>
```
Comment (WizardCM) [inline UI/forms/OBSBasic.ui:2281]: This translation key does not exist.

Comment (cg2121) [reply UI/forms/OBSBasic.ui]: This is now fixed.

**Comment by cg2121** [comment]
Closing because it was decided off thread that the transition toolbar wasn't the correct design for the transitions dock.

**Pull request closed without merging**